Tools/machines/lassen-llnl/install_v100_dependencies.sh

```diff
@@ -108,19 +108,25 @@ cmake --build ${build_dir}/lapackpp-lassen-build --target install --parallel 10
 
 # Python ######################################################################
 #
+# sometimes, the Lassen PIP Index is down
+export PIP_EXTRA_INDEX_URL="https://pypi.org/simple"
+
 python3 -m pip install --upgrade --user virtualenv
 rm -rf ${SW_DIR}/venvs/warpx-lassen
 python3 -m venv ${SW_DIR}/venvs/warpx-lassen
 source ${SW_DIR}/venvs/warpx-lassen/bin/activate
 python3 -m pip install --upgrade pip
 python3 -m pip cache purge
 python3 -m pip install --upgrade wheel
```

Comment:
https://github.com/h5py/h5py/issues/2337#issuecomment-1777771440
```suggestion
python3 -m pip install --upgrade wheel
python3 -m pip install --upgrade setuptools
```